utils/checks.stanza

```diff
@@ -747,16 +747,6 @@ public pcb-check generic-pin-check (p:JITXObject):
       fail-message =         "Generic pin %_ does not have the correct GenericPin property attached" % [ref(p)],
       locators =             [p]
     )
-    #CHECK(
-      condition =            has-property?(p.net-voltage),
-      name =                 "Generic pin checks"
-      description =          "Check net-voltage property on generic pin"
-      category =             "Pin Checks"
-      subcheck-description = "Verify the generic pin has a net-voltage property attached",
-      pass-message =         "Pin %_ has the net-voltage property (%_V) attached" % [ref(p), property(p.net-voltage)],
-      info-message =         "Pin %_ does not have the net-voltage property attached" % [ref(p)],
-      locators =             [p]
-    )
```

Comment:
What prompted the removal of this info check?

Reply:
It triggers on every pin with the generic pin property. `net-voltage` was meant for power rails so this was triggering a bunch of info checks that did not point towards improving the design

Reply:
Oh -- I assumed net-voltage was for more than just power nets (i.e. any net where the user wanted to check the voltage limit).

Reply:
Maybe we could bring it back, but make sure it's not trigger on unconnected pins?

Reply:
Well let's leave it off for now given the divergence in usage. Unconnected pins (with the no-connect property) were excluded from pin checks at one point. Not sure what the current status of property(pin.no-connect) is. For sure the checks.stanza looks for the no-connect property 
```
; Checks if a pin is marked "no-connect"
defn no-connect? (p:JITXObject) -> True|False : 
  has-property?(p.no-connect) or
  has-property?(p.nc) or 
  has-property?(p.NC)
```